packages/fgtclb/academic-base/.gitignore

```diff
@@ -0,0 +1,15 @@
+/.idea
```

Comment:
@sbuerk I'd also add the ".DS_Store" pattern here. Some extensions have the pattern, some others not. I only fixed that for EXT:academic_jobs in our pull request.
We can also make it in the next round.